src/app/(account)/login/page.tsx

```diff
@@ -10,8 +10,22 @@ export default function LoginPage() {
   const router = useRouter();
 
   useEffect(() => {
-    if (session) router.replace('/');
-  }, [session, router]);
+    const checkSession = async () => {
+      const { data } = await supabase.auth.getSession();
+      console.log(data);
```

Comment:
remove this log statement

Reply:
We need a way to keep some debug logging

Reply:
Create a ticket for this with low priority